src/components/groups-management/TaskGroupHeadPanel.js

```diff
@@ -46,6 +48,14 @@ class TaskGroupHeadPanel extends FormPanel {
       edited, classes, readOnly,
     } = this.props;
     const taskGroup = { ...edited };
+    const { rights } = this.props;
```

Comment:
The properties have already been destructured from _this.props_. Add _rights_ to the destructured properties.